.markdownlint-cli2.cjs

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const sharedStandards = require('@silvermine/standardization/.markdownlint-cli2.shared.cjs');
+
+module.exports = {
+
+   ...sharedStandards,
+
+   ignores: [
+      ...sharedStandards.ignores,
+      'CHANGELOG.md',
```

Comment:
This should already be ignored: https://github.com/silvermine/standardization/blob/v2.2.3/.markdownlint-cli2.shared.cjs#L15 Is something not working properly?